Sources/PayPalNativeCheckout/PayPalUIInterfaces/NativeCheckout.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import UIKit
+import PayPalCheckout
+#if canImport(PaymentsCore)
+import PaymentsCore
+#endif
+
+class NativeCheckoutProvider: NativeCheckoutStartable {
+
+    // swiftlint:disable function_parameter_count
```

Comment:
Do we have a follow up ticket to remove this rule?

Reply:
yep there's a ticket to create create/update lint rules